File: jbake-core/src/main/java/org/jbake/template/model/DBExtractor.java
package org.jbake.template.model;

import org.jbake.app.ContentStore;
import org.jbake.model.TemplateModel;

public class DBExtractor implements ModelExtractor<ContentStore> {

    @Override
    public ContentStore get(ContentStore db, TemplateModel model, String key) {
        return db;
    }

}

File: jbake-core/src/main/java/org/jbake/template/model/DBExtractor.kt
package org.jbake.template.model

import org.jbake.app.ContentStore
import org.jbake.template.TypedModelExtractor

class DBExtractor : TypedModelExtractor<ContentStore> {

    override fun extract(context: RenderContext, key: String): ContentStore {
        return context.db
    }
}
